utils/fake_libc_include/_fake_typedefs.h

```diff
@@ -2,7 +2,6 @@
 #define _FAKE_TYPEDEFS_H
 
 typedef int size_t;
-typedef int __builtin_va_list;
```

Comment:
Is this one duplicated too?

Reply:
Ah, no its not. Its being used to define a different macro. Thanks for catching my whoops. However, since `__builtin_va_list` ends up being parsed as a function call, it doesn't matter if its defined in the fake headers.

Reply:
Sorry, my bad again. I was mixing up `__builtin_va_list` with `__builtin_va_arg`... so many macros...